contents.json

```diff
@@ -17151,6 +17151,24 @@
       ],
       "date_added": "Nov 18 2018",
       "suggested_by": "@dkhamsing"
-    }
+    },
+    {
+      "title": "WeAre",
+      "category-ids": [
+        "photo"
+      ],
+      "description": "photo album of AR",
+      "source": "https://github.com/SherlockQi/HeavenMemoirs",
+      "homepage": "https://github.com/SherlockQi",
+      "itunes": "https://itunes.apple.com/app/id1234567890",		
```

Comment:
@SherlockQi https://itunes.apple.com/app/id1234567890 doesn't work for me. Could you change this link to a working link?